### ISSUE TITLE
Howdy!

### ISSUE THREAD

**Issue by smhk** [connorprzybyla/real-time-ios#2]
What about a super-simple, how-to!

Your code looks great, would love to try it.  Cheers.

**Comment by connorprzybyla** [comment]
Of course! Will have one posted soon in addition to a YouTube tutorial.
